src/core/components/odsCoreProvider/odsCoreProvider.mdx

```diff
@@ -37,7 +38,3 @@ export const Application = () => {
     )
 }
 ```
-
-Refer to the component's properties below for more details on the default configuration objects used for each provider.
-
-<ArgTypes of={OdsCoreProvider} />
```

Comment:
Why has this block been removed?

Reply:
It was an accident, fixed https://github.com/aragon/ods/pull/221/commits/213b022b7d7184c0d1624222ffd3e6e5e01530e0